### PR TITLE
DISCO-947 - RecommendedArtist carousel arrows only render if more content

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/DISCO-947
- Refactored original PR with help from @damassi to this PR
- This only adds this behavior to the `RecommendedArtist` component and carousel

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit bb51b1c03b27bd3ce07bccd546f1701e11869764

https://deploy-preview-2323--artsy-reaction.netlify.com

File: src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx [hunk 1]
```diff
@@ -144,6 +150,22 @@ const RecommendedArtist: FC<
...
+          const { Arrow, currentSlideIndex } = arrowProps
+          return (
+            <ArrowWrapper>{currentSlideIndex !== 0 && <Arrow />}</ArrowWrapper>
+          )
+        }}
+        renderRightArrow={arrowProps => {
+          const { Arrow, currentSlideIndex, slidesToScroll } = arrowProps
+          return (
+            <ArrowWrapper>
+              {currentSlideIndex < artistData.length - slidesToScroll && (
```
Comment (damassi) [inline src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx]: 👍

File: src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx [hunk 2]
```diff
@@ -84,6 +85,11 @@ const RecommendedArtist: FC<
   RecommendedArtistProps & { onArtworkClicked: () => void }
 > = ({ artist, onArtworkClicked }) => {
   const { user, mediator } = useContext(SystemContext)
+  const artistData = get(
+    artist,
+    a => a.artworks_connection.edges,
+    []
+  ) as object[]
```
Comment (damassi) [inline src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx]: 👍 thanks for moving the logic out of props

**Comment by kierangillen** [comment]
@pepopowitz I think that should be a separate PR as it may be a bit of a refactor of the carousel component. Going to merge this one for now.

**Comment by artsyit** [comment]
:rocket: PR was released in v16.3.1 :rocket:

**Pull request merged**